src/canvas-view/canvas-editor/canvas-editor.less

```diff
@@ -39,7 +39,7 @@
       vertical-align: middle;
     }
     .show-hide-connectors-label {
-      vertical-align: text-bottom;
+      vertical-align: 2px;
```

Comment:
Not for me?  Seems to bottom align.
![image](https://cloud.githubusercontent.com/assets/12733153/23081767/5635afb8-f524-11e6-80b0-5acdc93186f2.png)